prepare-module.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
In order to exit and clean up the work products in case any subcommand fails I would recommend setting following in the beginning of the file:
```
set -e
set -u
set -o pipefail
```

And specifying a trap function that does necessary clean up (for instance removing the `lambda-files` directory:

```
cleanup() {

  rm -rf lambda-files
  # Do other clean-up stuff here...

}
```

`trap cleanup 0`

Reply:
I am not sure we need a cleanup here, if everything goes as planned the `lambda-files` directory has been moved successfully to `tf_module` but if something fails you probably want the `lambda-files` directory to stick around so you can investigate the failure in more depth. There is another `npm run clean` that can be run manually to clean up the `lambda-files` if you don't need it any more.

Reply:
:+1: